packages/react-magma-dom/src/components/Select/Select.test.js

```diff
@@ -500,6 +504,124 @@ describe('Select', () => {
     expect(getByTestId('customClearIndicator')).toBeInTheDocument();
   });
 
+  describe('additional content', () => {
+    const helpLinkLabel = 'Learn more';
+
+    const onHelpLinkClick = () => {
+      alert('Help link clicked!');
+    };
+
+    it('Should accept additional content to the right of the select label', () => {
```

Comment:
This is currently only testing that it accepts additional content and that content is present, not anything related to its position